### ISSUE TITLE
Where to download Ninjam clients?

### ISSUE THREAD

**Issue by brylie** [justinfrankel/ninjam#4]
Where can Ninjam clients be downloaded? E.g. is Ninjam a VST plugin that can be integrated into a DAW?

In particular, I would like to download a client for Linux.

**Comment by strk** [comment]
A curses based client is in this repository, under ninjam/cursesclient directory. Has a Makefile (although it fails for me due to #5)

A commandline one is under ninjam/cmdclient--old/ and a gui one is under ninjam/guiclient/ (both without a Makefile)

A client which is not listed on the ninjam website is http://jamtaba.com/ (opensource)

**Comment by brylie** [comment]
Will the ninjam developer(s) be providing a Makefile for the UI?

**Comment by strk** [comment]
On Mon, Mar 16, 2020 at 12:38:08PM -0700, Brylie Christopher Oxley wrote:
> Will the ninjam developer(s) be providing a Makefile for the UI?

You may try that, start from the Makefile under cursesclient/ and
adapt it, if/when it works send a PR !

**Comment by JamieMcDonnell** [comment]
hi there, I've just started using ninjam with jamTaba, and I am very happy with it - thanks for your hard work on it ;) Can I ask where I can get ninjamCast for Windows 10? I only found one dead dropbox link to it ;( Thanks so much ;)

**Comment by patricksebastien** [comment]
Yes +1 for an official Makefile for ninjam/guiclient (not sure if it would work on linux / jack).

**Comment by antanasbruzas** [comment]
Hi, I've created a NINJAM client as a native Linux vst plugin. Available at https://github.com/antanasbruzas/abNinjam
Intention was to keep it simple and not overload it with many features that ninjam supports.

**Comment by mkschulze** [comment]
We created one here: https://github.com/mkschulze/JamWide
VST3/CLAP/AU

**Comment by brylie** [comment]
Thanks @mkschulze! Is there any chance of getting JamWide signed for macOS? I'm not able to get the plugin working despite trying to allow it via the system Privacy & Security settings.

**Comment by mkschulze** [comment]
Hi @brylie, I plan on doing it at some point. Currently I'm in the process of porting it to JUCE.

**Comment by mcascone** [comment]
> Thanks [@mkschulze](https://github.com/mkschulze)! Is there any chance of getting JamWide signed for macOS? I'm not able to get the plugin working despite trying to allow it via the system Privacy & Security settings.

New fan here! You can try `xattr -cr /path/to/ninjam.vst3` to remove the quarantine. Try reopening, likely you'll be blocked again. Go into System Prefs -> security and look for the listing for ninjam.vst3. Click "Open Anyway". Enter your password. 
Reload/Reopen the plugin one more time, it _should_ be allowed to work. I use Gig Performer, so in that app's Plugin Manager i `re-scan failed plugins` to enable the plugin.

**Comment by mkschulze** [comment]
> Thanks [@mkschulze](https://github.com/mkschulze)! Is there any chance of getting JamWide signed for macOS? I'm not able to get the plugin working despite trying to allow it via the system Privacy & Security settings.

starting with v 1.1 Beta 5, JamWide is signed now.
